### DISCUSSION TITLE
starlette.io website down / DNS broken

### DISCUSSION THREAD

**Discussion by Moritz-Schmidt** [encode/starlette#2260] [Potential Issue] [answered]
Hi there,

it seems like something is wrong with the DNS for starlette.io, when trying to open [https://www.starlette.io/](https://www.starlette.io/) I get a server not found error. I took a look at the DNS records and there currently are no A or AAAA records: [https://www.nslookup.io/domains/www.starlette.io/dns-records/](https://www.nslookup.io/domains/www.starlette.io/dns-records/) there are no records for starlette.io (without www. ) either.

**Comment by Kludex** [comment, marked as answer]
Yeah, I've contacted @tomchristie already. Unfortunately, there's nothing I can do. :(

**Reply by lovelydinosaur** [reply]
Yep. It should(?) be back up now, pending any DNS caching issues.

**Reply by Kludex** [reply]
Thanks Tom 🙏
